### PR TITLE
feat: support getting service

### PR DESCRIPTION
**Pull request by zccabb**

### CODE REVIEW

File: cmd/resource/get.go [hunk 1]
```diff
@@ -48,6 +48,17 @@ var (
...
 			return ssl
 		},
+		"service": func(id sdk.ID) interface{} {
+			cluster, err := cloud.DefaultClient.GetDefaultCluster()
+			if err != nil {
+				output.Errorf("Failed to get the default cluster: %s", err.Error())
+			}
+			service, err := cloud.DefaultClient.GetService(cluster.ID, id)
+			if err != nil {
+				output.Errorf("Failed to get ssl detail: %s", err.Error())
```
Comment (tokers) [inline cmd/resource/get.go:58]:
```suggestion
				output.Errorf("Failed to get service: %s", err.Error())
```

File: internal/cloud/api.go [hunk 2]
```diff
@@ -308,6 +308,20 @@ func (a *api) UpdateService(clusterID cloud.ID, config string) (*cloud.Applicati
 	return service, nil
 }
 
+func (a *api) GetService(clusterID cloud.ID, appID cloud.ID) (*cloud.Application, error) {
+
```
Comment (tokers) [inline internal/cloud/api.go:312]: Remove the empty line.

File: cmd/resource/get.go (hunk 1 above)
Comment (zccabb) [reply cmd/resource/get.go]: fixed

**Review by Hazel6869 — APPROVED** [review]

File: internal/cloud/api.go (hunk 2 above)
Comment (zccabb) [reply internal/cloud/api.go]: fixed

**Review by tokers — APPROVED** [review]

**Pull request merged**